services/wfm/src/main/java/org/openkilda/wfm/topology/nbworker/bolts/NeoOperationsBolt.java

```diff
@@ -52,7 +52,8 @@ protected void handleInput(Tuple input) {
         final String correlationId = input.getStringByField("correlationId");
         getLogger().debug("Received operation request");
 
-        try (Session session = driver.session(request.isReadRequest() ? AccessMode.READ : AccessMode.WRITE)) {
+        // todo (Nikita C): temporary solution to make all requests in read mode.
```

Comment:
Better throw exception on write session requests. In current state behavoiour become not obvious, and can make a trouble for person who will implement first "writable" handler.

Reply:
Agree, but I didn't implement correctly any way to define whether it request is in the read mode or in write. Will do it soon, after the release